README.md

```diff
@@ -22,11 +22,11 @@ As of now, when coupled with the CoronaTracker Firebase backend, the app anonymo
     * Add GoogleServices.json and GoogleServices-Info.plist to Android and iOS projects respectively.
 2. The App uses multiple Flutter plugins.
     * The `background_geolocator_flutter` plugin needs a license to be purchased in case of a release builds.
-    * Some sounds are played by this flutter plugin as debug but can be stripped off in relase builds.
+    * Some sounds are played by this Flutter plugin as debug, but can be stripped off in release builds.
```

Comment:
These debug sound fx controlled with `Config.debug`.  Set `debug: false` to disable sounds.

Reply:
Cool. I'll update the commit.